Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Dockerfile to build the application on Render
+FROM maven:3.8.5-openjdk-17 AS maven
+COPY /src /app/src
+COPY pom.xml /app/
+WORKDIR /app
+# Disable tests on Render
+RUN mvn clean package -DskipTests
+
+FROM eclipse-temurin:17-jre-alpine
+COPY --from=maven /app/target/*.jar /app/techpoker.jar
+EXPOSE 8080
```

Comment:
Doesn't this require an `ENTRYPOINT`, or does it automatically run any `.jar` in `/app/`?

Reply:
No. As mentioned in the Readme you define in the Render settings a Docker command. This will set the Entrypoint.

Reply:
Yeah, you can override on it on render, but wouldn't it make sense to have a runnable docker container, if you build locally? So a default `CMD java -jar /app/techpoker.jar` would make sense imho.

Reply:
I updated the dockerfile according to your hints. This is really a smarter way. Thanks. I wasn't familiar with starting using environment variables.